### PR TITLE
Add configuration for golang-migrate/migrate

### PR DESCRIPTION
**Pull request by zikani03**
Add entry for golang-migrate/migrate.

### CODE REVIEW

**Comment by alecthomas** [comment]
Sorry this took so long!

File: migrate.hcl [hunk 1]
```diff
@@ -0,0 +1,21 @@
+description = "Database migrations written in Go. Migrate reads migrations from sources and applies them in correct order to a database."
+test = "migrate --version"
+binaries = ["migrate"]
+
+version "v4.15.2" {
```
Comment (alecthomas) [inline migrate.hcl:5]: The version number can't include `v`. You'll have to insert the v below into the URLs.

**Review by alecthomas — CHANGES_REQUESTED** [review]

**Comment by zikani03** [comment]
No problem @alecthomas,  I have pushed the fix.

**Comment by alecthomas** [comment]
I think you also need to rebase to current master.

**Comment by zikani03** [comment]
@alecthomas done. Thought of rebasing when I realized the github-actions automatically upgrade (very cool btw), don't know how it slipped my mind.

**Comment by alecthomas** [comment]
Thanks!

**Pull request merged**